apis/apps/v1alpha1/node_pod_probe_types.go

```diff
@@ -58,7 +60,9 @@ type PodProbeStatus struct {
 	// pod uid
 	UID string `json:"uid"`
 	// pod probe result
-	ProbeStates []ContainerProbeState `json:"probeStates,omitempty"`
+	// +patchMergeKey=name
```

Comment:
This type is not needed because it is not issued externally.

Reply:
done